src/libPMacc/include/algorithms/math/floatMath/pow.tpp

```diff
@@ -43,6 +43,7 @@ struct Pow<float, float>
     HDINLINE result operator()(const float& base, const float& exponent)
     {
 #ifdef __CUDA_ARCH__ /*device version*/
+        /* @todo: CUDA has no pow(float, float) function yet but uses the C99 powf(float, float) */
```

Comment:
please let us keep it simple :)
`__cplusplus >= 201103L` and `just in CUDA 7.0` is not ideal as long as we support a broad spectrum of compilers and CUDA versions (what we should try to keep so).

probably adding that define and using undocumented functions is what I would refer to as "maintainer hell".